purefb/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # CHANGELOG - PureFB
 
+## 1.0.5 / 2024-06-26
```

Comment:
If we're sticking to SemVer and we have an **added** section, this should be `1.1.0`

Reply:
updated to 2.0.0